ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -233,7 +233,15 @@
       <WorkerDiagram::SingleFilter
         @egressFilter={{this.egressWorkerFilterEnabled}}
       />
-
+      <div class='target-worker-filter-explainer'>
+        <Hds::Form::HelperText>
+          {{#if this.egressWorkerFilterEnabled}}
+            {{t 'resources.target.workers.filter-explainer.egress-worker'}}
+          {{else}}
+            {{t 'resources.target.workers.filter-explainer.any-worker'}}
+          {{/if}}
+        </Hds::Form::HelperText>
+      </div>
```

Comment:
Could we move this logic into the WorkerDiagram::SingleFilter component? We can use the logical paths that are already defined in that component... and the DualFilter component when we get to that part.

Reply:
yep absolutely